ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml

```diff
@@ -1,9 +1,10 @@
 chain:
-  as: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision
+  as: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision
   steps:
   - chain: cucushift-installer-rehearse-aws-ipi-proxy-sdn-provision
   - ref: workers-rhel-aws-provision
   - ref: workers-rhel-pre-hook-sshtunnel
+  - ref: workers-rhel-pre-hook-fips
   - chain: workers-rhel
   env:
   - name: PLATFORM_VERSION
```

Comment:
This provision chain is dedicated for fips testing, so suggest to move `FIPS_ENABLED` env to the provision chain, but not set it on job config level.